simulation_parameters/microbe_stage/organelles.json

```diff
@@ -23,6 +23,9 @@
         "name": "pilus"
     },
     "rusticyanin": {
+        "productionColour": "#ad4a22",
+        "consumptionColour": "#ad4a22",
+        "iconPath": "res://assets/textures/gui/bevel/RusticyaninIcon.png",
```

Comment:
I think that these properties should be placed after the existing ones. Also "iconPath" is not clear enough, it should specify it is used for ATP balance showing.